packages/runtime/container-runtime/src/summarizer.ts

```diff
@@ -598,7 +599,7 @@ export class Summarizer extends EventEmitter implements ISummarizer {
         summaryCollection?: SummaryCollection,
     ) {
         super();
-        this.logger = ChildLogger.create(this.runtime.logger, "Summarizer");
+        this.logger = ChildLogger.create(this.runtime.logger, "Summarizer", { SummaryId: uuid() });
```

Comment:
SummarizerId vs. SummaryId? (Not sure myself).

Reply:
SummarizerInstanceId ?
Though looking at container.ts, we use 
                containerId: uuid(),
So I guess it's SummarizerId.

Reply:
i dislike containerId, and in fact bohemia overwrites it with their own value is some cases. I'd rather not use id, as i think it is misleading, maybe use the term tag instead, so SummarizerTag

Reply:
If we are switching to `Tag`, we should be consistent in other places though.

Reply:
Turns out this guid is 1-1 with the container's logging ID. Not very useful. Working on the counter of summaries in a container like Tony suggested - it already exists, just isn't stamped on all the events you'd want it to be.